src/fixup/fixup.cpp

```diff
@@ -71,6 +73,23 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   bool enable_mhd_floors = pin->GetOrAddBoolean("fixup", "enable_mhd_floors", false);
   bool enable_rad_floors = pin->GetOrAddBoolean("fixup", "enable_rad_floors", false);
   bool enable_rad_ceilings = pin->GetOrAddBoolean("fixup", "enable_rad_ceilings", false);
+  bool enable_phi_enforcement =
+      pin->GetOrAddBoolean("fixup", "enable_phi_enforcement", false);
+  params.Add("enable_phi_enforcement", enable_phi_enforcement);
+  Real enforced_phi = pin->GetOrAddReal("fixup", "enforced_phi", 0.);
```

Comment:
The default here probably shouldn't be zero.

Reply:
I think 0 is ok -- no net flux is a valid state, and I don't think there are really any special numbers > 0 (apart from the MAD limit but I don't think this machinery will work well there or above anyway because it'll produce extra flux that will hop off the black hole into the disk, and then machinery will generate more flux, etc. etc.)

Reply:
I'm just thinking is 0 the flux you want to preserve by default?

Reply:
I think zero is fine, its just saying "as small a flux as possible given the flux tuning parameters" which is sort of the only valid "special" value. This flux enforcement is off by default of course so I imagine it will only be used when someone has a specific flux to enforce in mind. 

The other option would be to not provide a default value at all for `enforced_phi` which I am fine with if you think that's better/safer.

Reply:
I would rather not provide a default and just die if it's on without a value set.

Reply:
Yeah I agree -- default value removed.